shell/config/version.js

```diff
@@ -0,0 +1,9 @@
+let _isRancherPrime = false;
+
+export function isRancherPrime() {
+  return _isRancherPrime;
+}
+
+export function setIsRancherPrime(v) {
+  _isRancherPrime = v.isRancherPrime === 'True';
```

Comment:
This does not match what the AP returns - should be property `RancherPrime` and the value is a lowercase string (but can we please to a case insensitive check ... also should accept a boolean value as well)